### PR TITLE
doxygen stuff

### PR DESCRIPTION
**Pull request by danielrking4290**

### CODE REVIEW

**Comment by danielrking4290** [comment]
Addresses https://github.com/WSU-CEG-6110-4410/ADAD-Craft/issues/107

**Review by fayu555 — COMMENTED** [review]
Nice work it looks pretty good to me

File: src/main.c [hunk 1]
```diff
@@ -2719,6 +2721,8 @@ void parse_command(const char *buffer, int forward)
     }
 }
 
+/// [issue] https://github.com/WSU-CEG-6110-4410/ADAD-Craft/issues/107
+//! The on_light() places or removes light at a location
```
Comment (fayu555) [inline src/main.c:2725]: Please check /// here

File: src/main.c [hunk 2]
```diff
@@ -918,6 +918,8 @@ int collide(int height, float *x, float *y, float *z)
     return result;
 }
 
+/// [issue] https://github.com/WSU-CEG-6110-4410/ADAD-Craft/issues/3
+//! collide_no_clip checks for collision on all axes except y. It is used to implement the ability to walk through walls
```
Comment (fayu555) [inline src/main.c:922]: please check line 922 /// here

File: src/map.h [hunk 3]
```diff
@@ -32,6 +32,7 @@ typedef struct {
     int dz;
     unsigned int mask;
     unsigned int size;
+    //! This is a test doxygen comment
```
Comment (fayu555) [inline src/map.h:35]: same here ///

**Comment by Letscet** [comment]
Looks good! Good work

**Pull request merged**